src/public/app/shared/animations/flip.animation.ts

```diff
@@ -0,0 +1,20 @@
+import { trigger, animate, transition, state, style } from '@angular/animations';
+
+export const flipAnimation = trigger('flipAnimation', [
+    transition(':enter', [
+        style({ transform: 'rotateY(0)' }),
+        animate('0.5s', style({ transform: 'rotateY(179.9deg)' })),
+    ]),
+    transition(':leave', [
+        style({ transform: 'rotateY(179.9deg)' }),
+        animate('0.5s', style({ transform: 'rotateY(0)' })),
+    ]),
+    // state('active', style({
```

Comment:
remove commented code?

Reply:
I left it so that we could use it later. I just didn't feel like writing it down somewhere else but I can remove it if you want.